packages/compass-shell/src/modules/history-storage.spec.ts

```diff
@@ -24,11 +24,11 @@ describe('HistoryStorage', function () {
 
   describe('#save', function () {
     it('creates the file and directory if not existing', async function () {
-      expect(async () => await fs.access(historyFilePath)).to.throw;
+      await fs.access(historyFilePath);
```

Comment:
Similarly here

Reply:
This was expected _to_ throw bc the history file shouldn't exist as a precondition of this test, then we call save and assert that fs.access does not throw, which yea just has to be awaited inline of the test, nothing really to assert, it is the assertion.